Tests/MimusTests/MatcherTests.swift

```diff
@@ -128,17 +128,17 @@ class FoundationMatcherTests: XCTestCase {
 
     func testIndexPathPassingInvocation() {
         let result = matcher.match(
-                expected: [IndexPath(row: 42, section: 43)],
-                actual: [IndexPath(row: 42, section: 43)]
+                expected: [IndexPath(item: 42, section: 43)],
```

Comment:
I assume this happens because we're now running tests as macOS bundled tests?

I don't think this is necessarily a problem, but perhaps we could use platform independent initialiser like `init(arrayLiteral: IndexPath.Element...)`?